tools/api_protolock/BUILD

```diff
@@ -0,0 +1,47 @@
+load("@rules_python//python:defs.bzl", "py_test")
+
+licenses(["notice"])  # Apache 2
+
+py_test(
+    name = "protolock_test",
+    srcs = ["protolock_test.py"],
+    data = [
+        "//tools/testdata/api_protolock:allowed/test_add_comment_current",
```

Comment:
It may be possible to have this list of testdata files in a build file in the testdata directory, and just reference that.